Process/PullRequests.md

```diff
@@ -36,9 +36,9 @@ Each PR will:
   2. Include a title that provides a one sentence overview of the purpose of the
      PR. Abbreviations can be used when necessary. Example: “PLAT-433 put the
      thumbnail under the original dir in the thumb path”.
-  3. Contain a description written using complete sentences of the changes being
+  3. Contain a description (or link to a description, i.e. Jira ticket) written using complete sentences of the changes being
      made and any context necessary to understand them. Example:
-     https://github.com/Wikia/vignette/pull/15.
+     https://github.com/Wikia/vignette/pull/15. Description field should not be empty, and linked resources must meet the above standards. 
```

Comment:
To be perfectly honest, I am not a huge fan of the changes I submitted in
the PR b/c I think jira tickets are almost always insufficient descriptions
for PRs. Jira tickets lay out requirements but PR descriptions discuss the
implementation itself. I find that the act of writing a PR description,
i.e. explaining in writing what I've just done, helps me make sure I
haven't missed anything and that my implementation is complete.

I think it would be great to leave the guidelines as they currently are and
leave it to the discretion of the engineer if they feel it's the right time
to "phone it in" by simply pasting a link to the jira ticket so there's at
least *something* in the description.

On Thu, Sep 10, 2015 at 4:17 PM, Aleksander Balicki <
notifications@github.com> wrote:

> In Process/PullRequests.md
> <https://github.com/Wikia/guidelines/pull/92#discussion_r39226338>:
>
> >       made and any context necessary to understand them. Example:
> > -     https://github.com/Wikia/vignette/pull/15.
> > +     https://github.com/Wikia/vignette/pull/15. Description field should not be empty, and linked resources must meet the above standards.
>
> what if jira link is in the title. can the description be empty then?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/guidelines/pull/92/files#r39226338>.
>



-- 
Liz Lee

FRONT END ENGINEER







*Videogames  /  Entertainment  /
 Lifestyle......................................................................360
3rd Street, Suite 750, San Francisco, CA 94107E liz@wikia-inc.com
<liz@wikia-inc.com>C 650-867-1913WEB WIKIA <http://wikia.com/> | FACEBOOK
<http://www.facebook.com/wikia> | TWITTER <http://twitter.com/wikia>*